Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -5033,6 +5033,9 @@ script:
   - arguments:
     - description: The serial number of the device.
       name: target
+    - defaultValue: running_config
+      description: Name of the file to save the config to. Default is "running_config"
```

Comment:
The default values are added to the README when it's generated, let's remove it from the yml

Reply:
I  addressed this in my new PR that I created.